app/models/cedar-metadata-template.ts

```diff
@@ -13,6 +14,16 @@ export default class CedarMetadataTemplateModel extends OsfModel {
     @attr('boolean') active!: boolean;
     @attr('object') template!: any;
     @attr('number') templateVersion!: number;
+
+    @tracked _recordCreated!: boolean;
```

Comment:
could likely make the getting/setting of this a little more streamlined, but not a blocker